source/_patterns/templates/typography/vertical-rhythm.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Vertical Rhythm
+---
+## Overview
+Vertical Rhythm offers a vertical flow for pages that have content that live outside of components. When the class .verticalRhythm is applied all typography level tags will receive a bottom margin. All headings will also be given the appropriate size and weight.
+
+## Usage
+
+| Class | Usage |
+| -- | -- |
+| `.pf-verticalRhythm` **Applied to:** wrapping `<div>`|  **Outcome:** applies top and bottom margins to all headings, paragraphs and lists. **Required:** No
```

Comment:
**Applied to:** `<div>`, `<section>` or `<article>`. 
what else can it be?
**Required:** Yes
It is required if you want to add vertical rhythm. The answer no is usually for modifiers.